android/app/src/androidTest/java/com/novoda/bonfire/user/UsersActivityTest.java

```diff
@@ -127,7 +127,7 @@ private void channelOwnersUpdateAfterEveryChange() {
         doAnswer(new Answer<Observable<DatabaseResult<Users>>>() {
             @Override
             public Observable<DatabaseResult<Users>> answer(InvocationOnMock invocation) throws Throwable {
-                return subject.asObservable().observeOn(AndroidSchedulers.mainThread());
+                return subject.observeOn(AndroidSchedulers.mainThread());
```

Comment:
Maybe it's worth using then [`.hide()`](http://reactivex.io/RxJava/2.x/javadoc/io/reactivex/Observable.html#hide()) as a replacement to `asObservable()`